extensions/devui/deployment/src/main/java/io/quarkus/devui/deployment/DevUIProcessor.java

```diff
@@ -462,30 +460,6 @@ void findAllJsonRPCMethods(BuildProducer<JsonRPCRuntimeMethodsBuildItem> jsonRPC
 
     }
 
-    @BuildStep(onlyIf = IsProduction.class)
-    void cleanProd(BuildProducer<RemovedResourceBuildItem> producer,
-            List<JsonRPCProvidersBuildItem> jsonRPCProvidersBuildItems,
-            CurateOutcomeBuildItem curateOutcomeBuildItem) {
-
-        List<RemovedResourceBuildItem> removedResourceBuildItems = new ArrayList<>();
-
-        for (JsonRPCProvidersBuildItem jsonRPCProvidersBuildItem : jsonRPCProvidersBuildItems) {
-
-            ArtifactKey artifactKey = jsonRPCProvidersBuildItem.getArtifactKey(curateOutcomeBuildItem);
-            if (artifactKey != null) {
-                removedResourceBuildItems.add(new RemovedResourceBuildItem(artifactKey,
-                        Set.of(jsonRPCProvidersBuildItem.getJsonRPCMethodProviderClass().getName())));
-            } else if (jsonRPCProvidersBuildItem.getJsonRPCMethodProviderClass().getName()
-                    .startsWith("io.quarkus.devui.runtime")
-                    || jsonRPCProvidersBuildItem.getJsonRPCMethodProviderClass().getName()
-                            .startsWith("io.quarkus.vertx.http.runtime")) {
-                removedResourceBuildItems.add(new RemovedResourceBuildItem(INTERNAL_KEY,
-                        Set.of(jsonRPCProvidersBuildItem.getJsonRPCMethodProviderClass().getName())));
-            }
-        }
-        producer.produce(removedResourceBuildItems);
-    }
```

Comment:
Can you please explain this change?

Reply:
Yes, this is just a cleanup that should have been removed when we moved dev-ui to it's own module. Now that module is excluded from prod mode, before that I had this as an alternative to "remove" at least the jsonrpc services from prod. We can keep it in as it does not do any harm. W.d.y.t ?

Reply:
Thanks, let's remove then